builtin/provisioners/chef/linux_provisioner.go

```diff
@@ -10,10 +10,27 @@ import (
 )
 
 const (
-	chmod      = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodLinux = "find %s -maxdepth 1 -type f -exec /bin/chmod %d {} +"
+	chmodAix   = "find %[1]s ! -path %[1]s -prune -type f -exec /bin/chmod %[2]d {} +"
 	installURL = "https://omnitruck.chef.io/install.sh"
 )
 
+func (p *provisioner) getChmodCommand() string {
```

Comment:
In Go getters are normally not prefixed by `set`:

```suggestion
func (p *provisioner) chmodCommand() string {
```

Reply:
Good to know. I'll fix this.